pumpkin-protocol/src/var_int_helper.rs

```diff
@@ -0,0 +1,206 @@
+use std::fmt::Debug;
```

Comment:
I elaborated on this, but I think it got lost in the sea of tabs.

When `var_int` and `var_long` is a single macro, it's meaningless as a file to keep around, and if someone is trying to understand how the type is defined, it's not super useful to have it in a separate file rather than putting them both in a single file. Nothing is lost and it's a bit easier to locate.

Reply:
Done